frontend/app/components/Playground.tsx

```diff
@@ -76,9 +80,13 @@ export const Playground = (props: PlaygroundProps) => {
             className="textarea textarea-bordered h-3/4"
             autoFocus
           />
-          <Button type="submit" disabled={isDisabled}>
-            Run
-          </Button>
+          {isShared ? (
```

Comment:
Will implement in a follow up PR. Requires a separate change on the backend. Doesn't break any current functionality